apps/web/src/app/api/v1/calendar/smart-scheduling/route.ts

```
@@ -0,0 +1,153 @@
+      return NextResponse.json(
+        { error: `Gemini API error: ${response.status}`, details: errorText },
+        { status: response.status }
+      );
+    }
+
+    const data = await response.json();
+    console.log('✅ Received response from Gemini API');
+    
+    // Kiểm tra phản hồi và trích xuất văn bản
+    if (data.candidates && data.candidates[0]?.content?.parts?.[0]?.text) {
+      const rawText = data.candidates[0].content.parts[0].text;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)